### PR TITLE
Applications backend and admin dashboard

### PR DESCRIPTION
**Pull request by lizfaria**
I am fetching real firebase data inside AdminDashboard component I created and then displaying the data using the Applist and AppListItems components. 

I set up redux functionality in AdminDashboard and actions/reducers folders, but not working in AdminDashboard file, so I commented out redux on that file and am just pulling it in directly with a request() function in the component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,35 +2,27 @@ import React, { useEffect } from "react";
 import "./App.css";
 import { request } from "./backend-request";
 import Dashboard from "./components/Dashboard"
+import { AdminDashboard } from "./pages/AdminDashboard"
+import { store } from "./redux/store";
```
Comment (sogolnaseri) [inline src/App.js]: could you please move store and provider to index.js?

File: src/App.js [hunk 2]
```diff
@@ -2,35 +2,27 @@ import React, { useEffect } from "react";
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Provider store={store}>
```
Comment (sogolnaseri) [inline src/App.js]: as I mentioned above its better to move  below lines to index.js.. thank you
ReactDOM.render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById("root")
);

File: src/redux/reducers/fetchApps.reducer.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      return {
+        ...state,
+        allApps: action.payload
+      };
+    default: {
+      return state;
+    }
+  }
+};
+
```
Comment (sogolnaseri) [inline src/redux/reducers/fetchApps.reducer.js]: if we are not using useEffect do you still want to keep these lines? I suggest you can remove it if we dont use it please

File: src/pages/AdminDashboard.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React, {useState, useEffect} from 'react';
+import { connect } from "react-redux";
+import { fetchAllApps, allApps, isLoading } from "../redux/actions/fetchApps.actions.js";
+import AppsList from "../components/AppsList.js"
+import { request } from "../backend-request";
+
+// export const AdminDashboard = ({isLoading, allApps, fetchAllApps}) => {
+export const AdminDashboard = () => {
+  const [allApps, setAllApps] = useState([])
+  // useEffect(() => {
```
Comment (sogolnaseri) [inline src/pages/AdminDashboard.js]: As I mentioned in some other places as well.. please remove commented out codes as we dont use those

File: src/pages/AdminDashboard.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    };
+    fetchData();
+  }, [])
+  console.log(allApps)
+  return (
+      <AppsList allApps={allApps} />
+  )
+}
+
+// const mapStateToProps = state => ({
```
Comment (sogolnaseri) [inline src/pages/AdminDashboard.js]: dont we need mapSatetToProps and mapDspatchToProps and connect?

File: src/App.js [hunk 6]
```diff
@@ -2,35 +2,27 @@ import React, { useEffect } from "react";
...
+import { store } from "./redux/store";
+import { Provider } from "react-redux";
 
 function App() {
   useEffect(() => {
     const fetchData = async () => { 
       const res = await request("applications")
       const data = await res.json();
-      console.log(data)
+      // console.log(data)
```
Comment (sogolnaseri) [inline src/App.js]: please remove unused code

File: src/App.js (hunk 1 above)
Comment (lizfaria) [reply src/App.js]: ok done!

File: src/App.js (hunk 2 above)
Comment (lizfaria) [reply src/App.js]: yes :)

**Pull request merged**